nurseconnect/templates/profiles/register.html

```diff
@@ -13,50 +13,28 @@ <h4 class="heading heading--subtitle">
             {% csrf_token %}
             <div class="FormGroup FormGroup--padded">
                 <div class="FormField FormField--center FormField--bordered">
-                    <label for="id_username">{% trans "Mobile Number" %}</label>
-                    {{ form.username.errors }}
+                    {{ form.username.label_tag }}
                     {{ form.username }}
-                    <span class="input-hint">({% trans "eg. 0821234567" %})</span>
+                    {{ form.username.errors }}
                 </div>
-
+                {{ form.non_field_errors }}
                 <div class="FormField FormField--center FormField--bordered">
-                    <label for="id_password">{% trans "Password" %}</label>
-                    {{ form.password.errors }}
+                    {{ form.password.label_tag }}
                     {{ form.password }}
-                    <span class="input-hint">({% trans "Enter password" %})</span>
+                    {{ form.password.errors }}
+                </div>
+                <div class="FormField FormField--center FormField--bordered">
+                    {{ form.confirm_password.label_tag }}
+                    {{ form.confirm_password }}
+                    {{ form.confirm_password.errors }}
                 </div>
-
-                {% if settings.profiles.UserProfilesSettings.show_mobile_number_field %}
-                    <div class="FormField FormField--center FormField--bordered">
-                        <label for="mobile_number">{% trans "Enter your mobile number" %}</label>
-                        {{ form.mobile_number.errors }}
-                        {{ form.mobile_number }}
-                        <span class="input-hint">{% trans "(e.g. +27900000013)" %}</span>
-                    </div>
-                {% endif %}
-
-                {% if settings.profiles.UserProfilesSettings.show_email_field %}
-                    <div class="FormField FormField--center FormField--bordered">
-                        <label for="id_email">{% trans "Enter your email addresss" %}</label>
-                        {{ form.email.errors }}
-                        {{ form.email }}
-                        <span class="input-hint">{% trans "(e.g. example@foo.com)" %}</span>
-                    </div>
-                {% endif %}
-
                 <div class="FormField FormField--center FormField--bordered">
-                    {% if form.terms_and_conditions.errors %}
-                        <span class="error">
-          {% trans "Please accept the T&amp;Cs in order to complete the registration" %}
-        </span>
-                    {% endif %}
                     {{ form.terms_and_conditions }}
-                    <label for="id_terms_and_conditions"
-                           class="profiles__terms">{% trans "I accept the Terms and Conditions" %}</label>
+                    {{ form.terms_and_conditions.label_tag }}
```

Comment:
Currently i’m unsure of how to address the terms of conditions link. Since CMS content is passed as context variables via the wagtailcore template tags. So that will take at least 2-3 hours. So for this PR, lets push it to the next QA release.